packages/documentation/pages/components/form.vue

```diff
@@ -81,18 +81,27 @@
 					<ComponentInfo :component="KtFormControllerObject" />
 					<h2>Personal Details</h2>
 					<KtFormControllerObject formKey="personalDetails">
-						<KtFieldText
-							formKey="firstName"
-							helpText="help for lastName"
-							label="First Name"
-							placeholder="Klaus"
-						/>
-						<KtFieldText
-							formKey="lastName"
-							helpText="help for lastName"
-							label="Last Name"
-							placeholder="Dieter"
-						/>
+						<template #default="{ setValue, values }">
+							<KtFieldText
+								formKey="firstName"
+								helpText="help for lastName"
+								label="First Name"
+								placeholder="Klaus"
+							/>
+							<KtFieldText
+								formKey="lastName"
+								helpText="help for lastName"
+								label="Last Name"
+								placeholder="Dieter"
+							/>
+							<KtFieldText
```

Comment:
I don’t have a better example in mind, but I think this doesn’t really fit the kind of intuitive example we wanted to have here